tests/unittests/test_handler/test_handler_growpart.py

```diff
@@ -130,7 +162,42 @@ def test_mode_auto_prefers_growpart(self):
             mockobj.assert_called_once_with(
                 ['growpart', '--help'], env={'LANG': 'C'})
 
-    @mock.patch.dict("os.environ", clear=True)
+    @mock.patch.dict("os.environ", {'LANG': 'cs_CZ.UTF-8'}, clear=True)
+    @mock.patch.object(temp_utils, 'mkdtemp', return_value='/tmp/much-random')
+    @mock.patch.object(stat, 'S_ISDIR', return_value=False)
+    @mock.patch.object(os.path, 'samestat', return_value=True)
+    @mock.patch.object(os.path, "join", return_value='/tmp')
+    @mock.patch.object(os, 'scandir', return_value=Scanner())
+    @mock.patch.object(os, 'mkdir')
+    @mock.patch.object(os, 'unlink')
+    @mock.patch.object(os, 'rmdir')
+    @mock.patch.object(os, 'open', return_value=1)
+    @mock.patch.object(os, 'close')
+    @mock.patch.object(shutil, 'rmtree')
+    @mock.patch.object(os, 'lseek', return_value=1024)
+    @mock.patch.object(os, 'lstat', return_value='interesting metadata')
+    def test_force_lang_check_tempfile(self, *args, **kwargs):
```

Comment:
Ooph, that's a lot of mocks. Not bad...but some of the base classes we don't use as often make a little more sense now :wink:

Reply:
I figured you might say that :). It was a good exercise in understanding "why" those test classes exist.